### PR TITLE
Plugin/sentence embedding v2

### PR DESCRIPTION
**Pull request by du-phan**

### CODE REVIEW

File: sentence-embedding/python-lib/dku_language_model/context_independent_language_model.py [hunk 1]
```diff
@@ -24,26 +23,29 @@ def get_sentence_word_vectors(self, text):
...
-
+    
     def compute_weighted_average_embedding(self, text, weights):
         """Weighted average embedding for computing SIF."""
-        embeddings = self.get_weighted_sentence_word_vectors(text, weights)
+        embeddings = self.get_weighted_sentence_word_vectors(text, weights)    
         avg_embedding = np.mean(embeddings, axis=0)
-        return avg_embedding.tolist()
-
+        return avg_embedding #.tolist()
```
Comment (JoachimZ) [inline sentence-embedding/python-lib/dku_language_model/context_independent_language_model.py]: dead code

File: sentence-embedding/custom-recipes/sentence-embedding-compute/recipe.py [hunk 2]
```diff
@@ -69,7 +70,7 @@
         embedded_texts = model.get_weighted_sentence_embedding(texts, smoothing_parameter, npc)
 
     # Checking for existing columns with same name
-    new_column_name = "{}_{}".format(name, aggregation_method)
+    new_column_name = "{}-{}".format(name, aggregation_method)
```
Comment (JoachimZ) [inline sentence-embedding/custom-recipes/sentence-embedding-compute/recipe.py]: sure? It breaks compatibility with previous version of the plugin!

File: sentence-embedding/python-lib/dku_language_model/context_independent_language_model.py (hunk 1 above)
Comment (du-phan) [reply sentence-embedding/python-lib/dku_language_model/context_independent_language_model.py]: removed!

File: sentence-embedding/custom-recipes/sentence-embedding-compute/recipe.py (hunk 2 above)
Comment (du-phan) [reply sentence-embedding/custom-recipes/sentence-embedding-compute/recipe.py]: this must have been a typo, I changed it back

**Pull request merged**